README.md

```diff
@@ -115,10 +113,11 @@ var editor = new MediumEditor('.editable', {
     toolbar: {
         /* These are the default options for the toolbar,
            if nothing is passed this is what is used */
-        buttons: ['bold', 'italic', 'underline', 'anchor', 'header1', 'header2', 'quote'],
+        buttons: ['bold', 'italic', 'underline', 'anchor', 'h1', 'h2', 'quote'],
         diffLeft: 0,
         diffTop: -10,
         firstButtonClass: 'medium-editor-button-first',
+        headerTags: ['h2', 'h3', 'h4'],
```

Comment:
If we agree we want to have 3 header buttons by default, I'll update the `buttons` array to include a 'h3' as well.  If we don't want 3 header buttons, I think we should leave this as-is, because these defaults don't necessarily have to be tied to each other.  A user could choose to enable all 3 header buttons, and if they do, it'd be nice if it worked by default (without requiring them to override the `headerTags` option).  What do you think?